src/BinSkim.Driver/AnalyzeOptions.cs

```diff
@@ -20,7 +20,7 @@ public class AnalyzeOptions : AnalyzeOptionsBase
             HelpText = "Execution traces, expressed as a semicolon-delimited list, that " +
                        "should be emitted to the console and log file (if appropriate). " +
                        "Valid values: PdbLoad.")]
-        public override IEnumerable<string> Traces { get; set; }
```

Comment:
>Traces [](http://example.com/codeflow?start=44&length=6)

this change comes from the sdk.
[https://github.com/microsoft/sarif-sdk/pull/2624](https://github.com/microsoft/sarif-sdk/pull/2624)